src/infrastructure/content/IContentCommunicator.java

```diff
@@ -0,0 +1,26 @@
+package infrastructure.content;
+
+public interface IContentCommunicator {
+	/*
+	 * UI will call this method with one json formatted string argument
+	 * @param : userDetails - It is json string which will contain server ipAddress, username, image
+	 */
+	void initialiseUser(String userDetails);
+	
+	/*
+	 * UI will call this to paas the message of client to everyone else
+	 * @param : message - it is the actual message as a string
+	 */
+	void sendMessageToContent(String message);
+	
+	/*
+	 * UI will call this to notify that the current user is quitting the session to all other users
+	 */
+	void notifyUserExit();
+	
+	/*
+	 * UI will subscribe to us in order to receive any updates coming from other users
+	 */
+	void subscribeForNotifications(String identifier, IContentNotificationHandler handler);

```

Comment:
Give the @param description here too. Put a @param tag on new line for each parameter